### PR TITLE
Model support for RTC feature

### PR DESCRIPTION
**Pull request by Ram81**
@deshraj @ayshrv @utsavgarg @kdexd this PR adds the required models for RTC feature support according to the db schema. It is dependent on #357.

### CODE REVIEW

File: ide/static/js/content.js [hunk 1]
```diff
@@ -755,7 +756,7 @@ class Content extends React.Component {
         },
         success : function (response) {
           if (response.result == 'success'){
-            var url = 'http://fabrik.cloudcv.org/caffe/load?id='+response.id;
+            var url = 'http://localhost:8000/caffe/load?id='+response.id;
             this.modalHeader = 'Your model url is:';
```
Comment (Ram81) [inline ide/static/js/content.js]: For testing purpose I have made it localhost right now, when moving to prod I'll update this.

**Comment by ayshrv** [comment]
@Ram81 I faced an error while setting this up.
```
reactjs    | ERROR in ./ide/static/js/content.js
reactjs    |
reactjs    | /code/ide/static/js/content.js
reactjs    |   127:5  error  Unexpected console statement  no-console
reactjs    |
reactjs    | ✖ 1 problem (1 error, 0 warnings)
```

**Comment by Ram81** [comment]
@ayshrv you can ignore that, it's just a console.log statement

**Comment by Ram81** [comment]
Closing this for now.

**Pull request closed without merging**